.github/workflows/mac.yml

```diff
@@ -7,6 +7,10 @@ on:
         required: false
         default: "release"
         type: string
+      wpt:
+        default: ""
+        required: false
+        type: string
       wpt-layout:
         required: false
         type: string
```

Comment:
This becomes a bit confusing with `wpt`, `wpt-layout`, and `wpt-sync` as it isn't clear what each of these does in an obvious way. Suggestion:

`wpt`: `wpt-tests-to-run`
`wpt-layout`: `wpt-layout-system`
`wpt-sync`: `wpt-sync-tests-from-upstream`